src/Agent/CHANGELOG.md

```diff
@@ -17,9 +17,13 @@ Enables nullable reference types that are part of the C# 8.0 language specificat
 * **Improved Support for NetTCP Binding in WCF Instrumation**
 When the NetTCP Binding type is used in Windows Communication Foundation (WCF), the Agent will now send and receive trace context information in support of Distributed Tracing (DT) or Cross Application Tracing (CAT).  Implements [#209](https://github.com/newrelic/newrelic-dotnet-agent/issues/209).
 
+* **.NET 5 Prerelease Support**<br/>
+Adds support for .NET 5 prerelease. 
+
 ### Fixes
 * Fixes an issue that may cause `InvalidCastException` due to an assembly version mismatch in Mvc3 instrumentation.
 * Fixes an async timing issue that can cause the end time of `Task`-returning methods to be determined incorrectly.
+* Fixes an issue that causes .NET 5 prerealease incompatability.
```

Comment:
We should be more specific and reference the issue in this changelog entry. For example, a .NET 5 app that does not use aspnetcore would have been just fine.